Assets/Scripts/RemoteInput.cs

```diff
@@ -16,14 +18,47 @@ enum EventType
         Mouse = 1,
         MouseWheel = 2,
         Touch = 3,
-        ButtonClick = 4
+        ButtonClick = 4,
+        Gamepad = 5,
+    }
+
+    enum GamepadEventType {
+      ButtonUp = 0,
+      ButtonDown = 1,
+      ButtonPressed = 2,
+      Axis = 3,
+    }
+
+    enum GamepadKeyCode {
+        Button0 = 0,
+        Button1,
+        Button2,
+        Button3,
+        Button4,
+        Button5,
+        Button6,
+        Button7,
+        Button8,
+        button9,
+        axis0Button,
```

Comment:
Maybe we can make the code consistent here, as in CamelCase, instead of both CamelCase and camelCase